shared/search/user-search/search-bar.desktop.js

```diff
@@ -69,18 +65,18 @@ class SearchBar extends Component<void, Props, void> {
     }
   }
 
-  render () {
-    const services = ['Keybase', 'Twitter', 'Facebook', 'Github', 'Coinbase', 'Reddit', 'Hackernews']
-    const tooltips: {[key: string]: ?string} = {'Hackernews': 'Hacker News'}
+  _services = ['Keybase', 'Twitter', 'Facebook', 'Github', 'Coinbase', 'Reddit', 'Hackernews']
+  _tooltips: {[key: string]: ?string} = {'Hackernews': 'Hacker News'}
```

Comment:
:sheep: Could move these out of the class entirely if they're not reused elsewhere.